Assets/Scripts/World/LettersToReid/CharacterCollisonHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class CharacterCollisonHandler : MonoBehaviour{
+	public MyCharacterController charactor;
+	public MemoryBehvior memory;
+	public bool memoryRead = false;
+	public int  enemyHits = 0;
+
+	Vector3[] memoryLocations = { new Vector3(5.76f, 0.26f, 0), new Vector3(16.21f, 2.37f, 0),
```

Comment:
These should not be hardcoded. Should detect memories using the `OnCollisionEnter` function